src/components/Hex/Hex.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react';
+
+function Hex(props) {
+    const [isCopied, setIsCopied] = useState(false);
+    const [textStyle, setTextStyle] = useState();
+
+    useEffect(() => {
+        if (isCopied) {
+            setTextStyle({
+                opacity: 0.2,
+                transition: "all 2s",
+            });
+            setTimeout(() => { setIsCopied(false); }, 1000);
```

Comment:
don't need to do this if you don't want to, but did you know you can do this?
```suggestion
            setTimeout(() => setIsCopied(false), 1000);
```

Reply:
That's cool, didn't know that!